images/dev.Dockerfile

```diff
@@ -0,0 +1,39 @@
+#FROM mcr.microsoft.com/devcontainers/go:0-1-bullseye
+FROM almalinux:8
```

Comment:
We should pin the platform as arm here, otherwise this Dockerfile (which is tailored for arm) will be built using the GHA runner's host OS -- most definitely amd64.